src/aruco_pose/src/aruco_pose.cpp

```diff
@@ -161,6 +139,20 @@ void ArucoPose::image_raw_callback(
         // Median filter gets applied
         median_filters_map_[id]->update(raw_rpyxyz, median_filtered_rpyxyz);
 
+        // Add a tf to map the camera base to a hypothetical link in front of the camera
+        geometry_msgs::msg::TransformStamped transformStamped_map;
```

Comment:
does this "map" reference frame, have a better technical name (that would be broadly recognized) like "image_frame", "CV_frame" or something to that extent?

Reply:
Changed the name of 'map' to 'image_reference_frame'.
https://github.com/maffettone/erobs/blob/01dec49d475b66d9f6bcae9cbe1f5dcb73c92be9/src/aruco_pose/src/aruco_pose.cpp#L143